i18n.ts

```diff
@@ -43,3 +48,55 @@ function getLanguageCode(code: string) {
   const [language] = code.split('-');
   return language;
 }
+
+export function getVCDetailsForCurrentLanguage(locales) {
```

Comment:
can this function be refactored?

Please separate the nested logic into following:

1. check if supportedLanguage has current Language
2. if it has, get VC Details for current Language
3. if it doesn't, return safe value(I assume its returning undefined now)